.github/workflows/ci.yml

```diff
@@ -104,8 +104,8 @@ jobs:
           HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: 1
         run: |
           # A workaround for "The `brew link` step did not complete successfully" error.
-          brew install python@3 || brew link --overwrite python@3
-          brew install automake libtool pkg-config gnu-getopt ccache boost libevent miniupnpc libnatpmp zeromq qt@5 qrencode
+          brew install --quiet python@3 || brew link --overwrite python@3
```

Comment:
Are these Qt workarounds still needed? Would be good if we could drop them, rather than adding more code to suppress warnings coming from workarounds that may no-longer be needed.